### PR TITLE
Add MCEP100, MCEP098, MCF003, Change to banners and some links

### PR DESCRIPTION
**Pull request by GiovanniMCMXCIX**
Demo at https://giovannimcmxcix.github.io/monstercat-browser/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,6 +211,12 @@
 				var card_title = albumData.seasonals[albumIndex2].album_artist;
 				var card_subtitle = albumData.seasonals[albumIndex2].album_title;
 			}
+			else if(number.indexOf("MCF") > -1 || number.indexOf("MCX") > -1) {
```
Comment (gamehelp16) [inline index.html:214]: Why not combine this with the previous if statement?

File: albums.js [hunk 2]
```diff
@@ -989,8 +989,23 @@ var albumData = {
 			"album_title": "Synchronize (VIP Mix / Acoustic) EP",
 			"album_artwork_thumb": "https://i.imgur.com/S1uMtsMm.jpg",
 			"album_artwork_full": "https://i.imgur.com/S1uMtsM.jpg"
+		},
+
+		{
+			"album_number": "MCEP098",
```
Comment (gamehelp16) [inline albums.js:995]: The albums here need to be listed in chronological order in order for it to appear correctly on the site.

EDIT: Also while you're at it, would you please also switch the orders of MCX002 and MCH001? Thank you in advance!

File: index.html [hunk 3]
```diff
@@ -346,7 +346,7 @@
 
 				if(includeMix) {
 					for(z=0;z<data.mixes.length;z++) {
-						var thetrack = z+90;
+						var thetrack = z+100;
```
Comment (gamehelp16) [inline index.html]: What is the reason behind this change?

**Comment by gamehelp16** [comment]
Sorry I didn't realize that you also did a PR instead of just submitting an issue, anyways here are some things:
1. Why do you decide to use a .png file instead of .svg for the monstecat logo?
2. The indentations on several files (mainly on the new album data files you made) isn't quite right. I know this actually doesn't really matter but it bugs me quite a lot.
3. Even though those seasonal albums are indeed made by various artists, but tbh I feel like putting "Monstercat" as the artist is better even though it's not made by Monstercat, like the compilation albums.
4. I like the new banners! However tbh I prefer the old banner for the EPs because it fits more with the dark theme of the site, the other banners are awesome though!

There are also a few other comments that I have "tied" with some certain files which you can read above. Thank you for your effort to help with the site!

**Comment by gamehelp16** [comment]
Oh, also it looks like you're editing only using the web version of GitHub and it looks like pain in the butt. I recommend you to download some git softwares (not really sure what to call them) so that you can actually review your changes before committing them!

**Comment by GiovanniMCMXCIX** [comment]
I saw that I messed up the indentations in some files and I'll change them, the .svg had the logo looked a bit off on the menu bar, some album numbers (or catalog id) are not in the exactly in the release date example Hollow Mind is MCEP098 and i was released after Synchronize (VIP Mix / Acoustic) EP that is MCEP099 and that applies to many releases and I'll close this pull request and send another one after I made most of the fixes

**Comment by gamehelp16** [comment]
> the .svg had the logo looked a bit off on the menu bar

Do you by any chance have the .svg version of the correct logo? Tbh I prefer .svg much more than .png in this case.

> some album numbers (or catalog id) are not in the exactly in the release date

whoa, didn't know about that before!
